habitat/hooks/run

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+export GEM_HOME="$(hab pkg path chef/chef-client)/ruby/2.3.0"
+export GEM_PATH="$(hab pkg path core/ruby)/lib/ruby/gems/2.3.0:$(hab pkg path core/bundler):$GEM_HOME"
```

Comment:
⚠️  These might pull in versions of `core/ruby` or `core/bundler` that do not match the version against which the package was built. Related: the [RFC for setting some runtime environment variables at build time](https://github.com/habitat-sh/habitat/issues/1579).